### DISCUSSION TITLE
Integrating AS & Country Analysis: IODA Alarms in Network Reports

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#31] [General]
Hey @romain-fontugne ,

We now have a [reusable IODA Alarms component](https://github.com/InternetHealthReport/ihr-website/blob/master/src/views/charts/IodaChart.vue). How about configuring it within the Network Reports Page to analyze AS and Country granularity? Analyzing the country granularity would provide insights into the specific contribution of a particular AS to the overall country traffic.

If this idea seems promising, where do you think would be the best place to position this within the network report page for logical ordering, considering our current sections:
1. AS Dependency
2. Route Origin Validation
3. Network Delay
4. Link Delay and Forwarding Anomalies
5. Network Disconnections

**Comment by romain-fontugne** [comment]
Yes, this is a good idea and also would love to add Cloudflare Radar's traffic data in the network reports. I'm not so sure about the order of the sections. Maybe between 1 and 2 or between 2 and 3, makes more sense to me

**Reply by moawnallah** [reply]
Cloudflare Radar's traffic data seems interesting:
https://radar.cloudflare.com/traffic/jp

For double-checking this is the API we wanna extract (on the AS granularity):
https://developers.cloudflare.com/api/operations/radar-get-traffic-anomalies
 
I'm gonna address IODA Chart & Cloudflare Radar in the Network Report after submitting the PR for Global Report V2:
https://github.com/InternetHealthReport/ihr-website/pull/669#issuecomment-1861087781
